packages/flutter/lib/src/material/theme_data.dart

```diff
@@ -726,6 +736,7 @@ class ThemeData extends Diagnosticable {
     PageTransitionsTheme pageTransitionsTheme,
     AppBarTheme appBarTheme,
     BottomAppBarTheme bottomAppBarTheme,
+    FloatingActionButtonThemeData floatingActionButtonTheme,
```

Comment:
Put this under `buttonColor` or under `dialogTheme` so it's kinda alphabetical.

Reply:
Done